### PR TITLE
Content

### PR DESCRIPTION
**Pull request by rhysfaultless-cpr**
Deployed on AWS for review:
https://content-accessories.dfy90wyu8dics.amplifyapp.com/

### CODE REVIEW

File: docs/accessories/index_accessories.mdx [hunk 1]
```diff
@@ -5,17 +5,21 @@ sidebar_position: 1
 ---
 
 <center>
-  <img
-    src="/img/about_images/about_1.jpg"
-    width="1000"
-  />
+  <figure>
+    <a href="https://norlab.ulaval.ca/">
```
Comment (jhiggins-cpr) [inline docs/accessories/index_accessories.mdx]: Are we sure that we want to be linking to third-party pages from here?

File: docs/accessories/network_switch.mdx [hunk 2]
```diff
@@ -5,10 +5,94 @@ sidebar_position: 6
...
+
+| ID  | Description                                                    |                         CPR item                         | Quantity |
+| :-: | :------------------------------------------------------------- | :------------------------------------------------------: | :------: |
+|  1  | Kit, Attachment—80 X 80, DIN Rail                              |           [027881](/docs/accessories/din_rail)           |    1     |
+|  2  | Network Switch, Ethernet, RJ45—TRENDnet TI-G50, 5 Port Gigabit | [021044](/assets/pdf/clearpath_robotics_021044-TDS1.pdf) |    1     |
+|  3  | Spacer, DIN rail, Locking—35 X 7.5 X 10                        |                          023649                          |    2     |
+
+### Instructions, 5 Port
+
+1.  Build the DIN rail assembly. The indructions are listed on the [DIN Rail page](/docs/accessories/din_rail)
```
Comment (jhiggins-cpr) [inline docs/accessories/network_switch.mdx]: "instructions"

File: docs/accessories/network_switch.mdx [hunk 3]
```diff
@@ -5,10 +5,94 @@ sidebar_position: 6
...
+
+| ID  | Description                                                    |                         CPR item                         | Quantity |
+| :-: | :------------------------------------------------------------- | :------------------------------------------------------: | :------: |
+|  1  | Kit, Attachment—80 X 80, DIN Rail                              |           [027881](/docs/accessories/din_rail)           |    1     |
+|  2  | Network Switch, Ethernet, RJ45—TRENDnet TI-G50, 5 Port Gigabit | [025332](/assets/pdf/clearpath_robotics_021044-TDS1.pdf) |    1     |
+|  3  | Spacer, DIN rail, Locking—35 X 7.5 X 10                        |                          023649                          |    2     |
+
+### Instructions, 8 Port
+
+1.  Build the DIN rail assembly. The indructions are listed on the [DIN Rail page](/docs/accessories/din_rail)
```
Comment (jhiggins-cpr) [inline docs/accessories/network_switch.mdx]: "instructions"

File: docs/accessories/usb_hub.mdx [hunk 4]
```diff
@@ -1,4 +1,51 @@
...
+
+| ID  | Description                             |                         CPR item                         | Quantity |
+| :-: | :-------------------------------------- | :------------------------------------------------------: | :------: |
+|  1  | Kit, Attachment—80 X 80, DIN Rail       |           [027881](/docs/accessories/din_rail)           |    1     |
+|  2  | USB Hub—7 Port, USB 3.0                 | [025440](/assets/pdf/clearpath_robotics_025440-TDS1.pdf) |    1     |
+|  3  | Spacer, DIN rail, Locking—35 X 7.5 X 10 |                          023649                          |    2     |
+
+### Instructions, 5 Port
+
+1.  Build the DIN rail assembly. The indructions are listed on the [DIN Rail page](/docs/accessories/din_rail)
```
Comment (jhiggins-cpr) [inline docs/accessories/usb_hub.mdx]: "instructions"

File: docs/components/fasteners/nut.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (jhiggins-cpr) [inline docs/components/fasteners/nut.mdx:1]: I didn't realize that we were adding all fasteners here. Does this duplicate any information that is already available somewhere else internally? Or is this intended to be the public facing version of your parts spreadsheet?

File: docs/computers/index_computers.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  </figure>
+</center>
+
+<br />
+
+Clearpath's robots use ROS or ROS2, which require the robot to have a computer running an Operating System like Ubuntu.
+We mostly use mini-ITX computers with Intel Core i3 and i7 processors, but we can run ROS on the major architectures including:
+
+- Intel Core i series
+- AMD Ryzen series
```
Comment (jhiggins-cpr) [inline docs/computers/index_computers.mdx]: Have we tried this anytime recently? Mostly just curious. Agreed that it should work.

File: docs/tools/robot_troubleshooting.mdx [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+Refer to the [Physical Tools page](#common_hand_tools) for a list of tools that you may need to access the electronics inside your robot.
+
+## Physical Tools
+
+- Multimeter
+- RJ45 Network cable
+- Any converters for your development computer, to connect an RJ45 Network cables
+- Monitor with a HDMI port
+- HDMI cable
+- Keyboard with a teathered USB cable (a mouse is not required)
```
Comment (jhiggins-cpr) [inline docs/tools/robot_troubleshooting.mdx]: "tethered"

**Review by jhiggins-cpr — CHANGES_REQUESTED** [review]
Just a few minor typo corrections, then ready to approve

File: docs/accessories/index_accessories.mdx (hunk 1 above)
Comment (rhysfaultless-cpr) [reply docs/accessories/index_accessories.mdx]: I will change this to a related norlab blog post on clearpathrobotics.com/blog

File: docs/components/fasteners/nut.mdx (hunk 5 above)
Comment (rhysfaultless-cpr) [reply docs/components/fasteners/nut.mdx:1]: This is a public facing version of our common parts. I think this information can help our customers quickly find replacement parts, or help with their integrations. We can also simplify our User Manuals by redirecting readers to the Components area of the website.

Should we keep the Components section of the website?

File: docs/computers/index_computers.mdx (hunk 6 above)
Comment (rhysfaultless-cpr) [reply docs/computers/index_computers.mdx]: I'm not aware of any AMD builds in the past year. This was a more common request ~3 years ago when the Ryzen line was notably faster than Intel.

Should AMD be removed from this page?

File: docs/components/fasteners/nut.mdx (hunk 5 above)
Comment (jhiggins-cpr) [reply docs/components/fasteners/nut.mdx:1]: I'm not opposed to sharing fasteners as long as it doesn't become a maintenance issue over time. If the list is fairly static, then it shouldn't be a problem.

File: docs/computers/index_computers.mdx (hunk 6 above)
Comment (jhiggins-cpr) [reply docs/computers/index_computers.mdx]: Maybe let's remove AMD for now

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**